README.md

```diff
@@ -0,0 +1,27 @@
+# RideMetricSampleApp
+This is a sample app uses RideMetric SDK and can be expanded to include specific functionality and features.
+
+# RideMetric SDK
+
+## RideMetric SDK configuration
+Init SDK
+
+```
+RideMetricSDK.initRidemetricApps(applicationContext);
+```
+Register user and listeners
+```
+RideMetric.registerUser(
+    activity,
+    tripListener, // can be null
+    registerListener // can not be null
+);
+```
+TripListener listener will be called to notify trip events.
+
+## Instalation
+
+Gradle
+```
+compile 'com.ridemetric:ridemetric-controlsdk:2.1.25'
```

Comment:
Don't forget to add the com.ridemetric:ridemetric-controlsdk:2.1.25 to the public repo